### DISCUSSION TITLE
Android: List scrolled back to top when adding to a list

### DISCUSSION THREAD

**Discussion by OscarCreator** [community/community#36868] [Other Feature Feedback, Questions, & Ideas]
When I scroll far down in the list of starred repos so the app have to fetch more items, and then add one of these repos to a list. The list is scrolled to the top again.

**Comment by MEDVEDEFFIZZAMUZZIC1996** [comment]
> When I scroll far down in the list of starred repos so the app have to fetch more items, and then add one of these repos to a list. The list is scrolled to the top again.

**Comment by Patek0000** [comment]
> When I scroll far down in the list of starred repos so the app have to fetch more items, and then add one of these repos to a list. The list is scrolled to the top again.

**Comment by Patek0000** [comment]
> > When I scroll far down in the list of starred repos so the app have to fetch more items, and then add one of these repos to a list. The list is scrolled to the top again.
> 
>

**Comment by kude5n1k** [comment]
Android studio or flutter?
Sample code?

**Comment by twiddler** [comment]
See #20353.

**Comment by Professormalik** [comment]
Software engineer

**Comment by Harforlarbiey** [comment]
Can't login my account

**Comment by EasonEatEgg** [comment]
您好，您的邮件我已经收到~尽快给您回复。                                                                                                         肖意成

**Comment by Loveyoumyfamily** [comment]
Yes

**Comment by Loveyoumyfamily** [comment]
Yes

**Comment by Ada1433** [comment]
Yes
